### ISSUE TITLE
Adding widgets mostly impossible 

### ISSUE THREAD

**Issue by durcheinandr** [mpcjanssen/simpletask-android#221]
I have some problems with adding widgets to my Homescreen (recent added widgets work).
System: Android 5.0.2
Version: Simpletask Cloudless 5.3.1 (from Play Store)

Steps to reproduce:

* add widget - select any other tab than starting tab (sort) -> force close

Also the existing widgets didn't show the *due:date*-field anymore.

**Comment by mpcjanssen** [comment]
I will rewrite this part of the code shortly, but did you already raise this issue via play? E.g. did you press the report error button when Simpletask crashed?

**Comment by durcheinandr** [comment]
I pressed report, but phone stated that there was an error.

**Comment by mpcjanssen** [comment]
Do you have experimental script support enabled?
If so what happens if you disable it?

**Comment by durcheinandr** [comment]
If I disable script support (Use Lua) same behavior (force close)

**Comment by mpcjanssen** [comment]
Could you try with one of the builds at https://mpcjanssen.nl/files/simpletask/latest/

**Comment by durcheinandr** [comment]
Didn't change anything with widget creation (force close)

**Comment by mpcjanssen** [comment]
Would it be possible to create a logcat? I was fairly sure the builds in that dir should have fixed the issue.

**Comment by durcheinandr** [comment]
logcat (filtered for simpletask):

```
04-22 17:35:08.177: I/ActivityManager(854): Start proc nl.mpcjanssen.simpletask for broadcast nl.mpcjanssen.simpletask/.MyAppWidgetProvider: pid=22743 uid=10506 gids={50506, 9997, 1028, 1015, 1023} abi=armeabi-v7a
04-22 17:35:08.264: I/Timeline(3524): Timeline: Activity_launch_request id:nl.mpcjanssen.simpletask time:6502538
04-22 17:35:08.265: I/ActivityManager(854): START u0 {act=android.appwidget.action.APPWIDGET_CONFIGURE cmp=nl.mpcjanssen.simpletask/.FilterActivity (has extras)} from uid 10519 on display 0
04-22 17:35:08.369: V/FilterActivity(22743): Called with intent: Intent { act=android.appwidget.action.APPWIDGET_CONFIGURE cmp=nl.mpcjanssen.simpletask/.FilterActivity (has extras) }
04-22 17:35:08.420: V/FilterActivity(22743): Created new fragment: nl.mpcjanssen.simpletask.FilterListFragment
04-22 17:35:08.430: V/FilterActivity(22743): Created new fragment: nl.mpcjanssen.simpletask.FilterSortFragment
04-22 17:35:08.759: I/ActivityManager(854): Displayed nl.mpcjanssen.simpletask/.FilterActivity: +464ms
04-22 17:35:09.077: I/Timeline(854): Timeline: Activity_windows_visible id: ActivityRecord{24294c11 u0 nl.mpcjanssen.simpletask/.FilterActivity t549} time:6503351
04-22 17:35:09.881: I/AppTracker(3308): nl.mpcjanssen.simpletask opened at 1429716909880
04-22 17:35:13.268: V/FilterActivity(22743): Created new fragment: nl.mpcjanssen.simpletask.FilterOtherFragment
04-22 17:35:13.271: E/AndroidRuntime(22743): Process: nl.mpcjanssen.simpletask, PID: 22743
04-22 17:35:13.271: E/AndroidRuntime(22743): 	at nl.mpcjanssen.simpletask.FilterActivity$MyTabsListener.onTabSelected(FilterActivity.java:484)
04-22 17:35:13.273: W/ActivityManager(854): Process nl.mpcjanssen.simpletask has crashed too many times: killing!
04-22 17:35:13.273: W/ActivityManager(854):   Force finishing activity nl.mpcjanssen.simpletask/.FilterActivity
04-22 17:35:13.286: W/ActivityManager(854): Wtf, activity ActivityRecord{24294c11 u0 nl.mpcjanssen.simpletask/.FilterActivity t549 f} in proc activity list not using proc ProcessRecord{2d2efd86 22743:nl.mpcjanssen.simpletask/u0a506}?!?
04-22 17:35:13.311: I/ActivityManager(854): Killing 22743:nl.mpcjanssen.simpletask/u0a506 (adj 1): crash
04-22 17:35:13.588: I/ActivityManager(854): Start proc nl.mpcjanssen.simpletask for restart nl.mpcjanssen.simpletask: pid=22785 uid=10506 gids={50506, 9997, 1028, 1015, 1023} abi=armeabi-v7a
04-22 17:35:13.651: W/ActivityManager(854): Spurious death for ProcessRecord{2d2efd86 22785:nl.mpcjanssen.simpletask/u0a506}, curProc for 22743: null
04-22 17:35:14.732: I/AppTracker(3308): nl.mpcjanssen.simpletask closed at 1429716914732
```

**Comment by mpcjanssen** [comment]
Please recheck with 5.3.2 if still crashing can you mail me a full log cat so without filtering?

**Comment by durcheinandr** [comment]
I rechecked with the build from https://mpcjanssen.nl/files/simpletask/latest/ and now it works again. Thanks

**Comment by mpcjanssen** [comment]
Great thanks for rechecking. The fix included there is part of 5.3.2 so closing
